toolkits/x/arcade_x/tools/tweets.py

```diff
@@ -63,17 +64,25 @@ async def search_recent_tweets_by_username(
     context: ToolContext,
     username: Annotated[str, "The username of the X (Twitter) user to look up"],
     max_results: Annotated[
-        int, "The maximum number of results to return. Cannot be less than 10"
+        int, "The maximum number of results to return. Must be in range [1, 100] inclusive"
     ] = 10,
+    next_token: Annotated[
+        Optional[str], "The pagination token starting from which to return results"
+    ] = None,
```

Comment:
Do we need to explain anything here to reduce the likelihood of the LM hallucinating a value for this?

Reply:
From my experience, even for older models, it errs on the side of caution and will either predict the `next_token` param correctly, or it doesn't provide the token when it should.